### PR TITLE
#86 Top of book graph

### PR DESCRIPTION
**Pull request by Lercerss**
#111, #108, #155 Implementation for the Top of Book graph

- Database schema modifications require dropping existing volumes
- New backend route `/topofbook/start/end/num_points`
- New frontend component `TopOfBookGraph` which requests points on load

### CODE REVIEW

File: app/src/styles/TopOfBookGraphWrapper.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const Styles = {
+};
```
Comment (chimano) [inline app/src/styles/TopOfBookGraphWrapper.ts]: Can this be removed?

File: core/graphelier-service/db/mongo.go [hunk 2]
```diff
@@ -75,11 +78,19 @@ func NewConnection() (*Connector, error) {
 func (c *Connector) GetOrderbook(instrument string, timestamp uint64) (result *models.Orderbook, err error) {
 	defer utils.TraceTimer("mongo/GetOrderbook")()
 
+	meta, ok := c.Cache.Meta[instrument]
+	if !ok {
+		return nil, InstrumentNotFoundError{Instrument: instrument}
+	}
+	interval := meta.Interval
+
+	exactTimestamp := (timestamp - (timestamp % interval)) / interval
```
Comment (chimano) [inline core/graphelier-service/db/mongo.go]: Can you refactor this into a function?

File: app/src/styles/TopOfBookGraph.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export const Styles = {
+};
```
[the remark this answers is not in the document]
Comment (panoreak) [reply app/src/styles/TopOfBookGraph.ts]: Indeed it can

File: app/src/styles/TopOfBookGraphWrapper.ts (hunk 1 above)
Comment (panoreak) [reply app/src/styles/TopOfBookGraphWrapper.ts]: This one as well

**Pull request merged**